src/Valit/Rules/ValitRule.cs

```diff
@@ -44,24 +41,18 @@ private ValitRule()
         }		
 
 		void IValitRuleAccessor<TObject, TProperty>.SetPredicate(Predicate<TProperty> predicate)
-		{
-			_predicate = predicate;
-		}
+            => _predicate = predicate;
 
 		void IValitRuleAccessor.AddError(ValitRuleError error)
-		{
-			_errors.Add(error);
-		}
+            => _errors.Add(error);
 
 		void IValitRuleAccessor<TObject, TProperty>.AddCondition(Predicate<TObject> condition)
```

Comment:
why this one is not replaced with "goes to " operator :d

Reply:
got me...